languages/python/TreeSitter/Python.hs

```diff
@@ -18,5 +18,5 @@ $(do
   currentFilename <- loc_filename <$> location
   pwd             <- runIO getCurrentDirectory
   let invocationRelativePath = takeDirectory (pwd </> currentFilename) </> "../vendor/tree-sitter-python/src/node-types.json"
-  input <- liftIO (eitherDecodeFileStrict' invocationRelativePath)
+  input <- runIO (eitherDecodeFileStrict' invocationRelativePath)
```

Comment:
In GHC 8.6 `Q` is no longer an instance of `MonadIO`, but you can use `runIO` to the same effect.

Reply:
TIL. I wonder why!